regolith/tools.py

```diff
@@ -2315,3 +2315,42 @@ def get_appointments(person, appointments, target_grant=None):
             appointments.append((person.get('_id'), bd, ed, appt.get('loading'),
                          round(weighted_duration, 2), appt.get('grant')))
     return appointments
+
+def add_todo_id_to_tasks(gtx, client, db, coll, milestone_id, task_id):
+    pdoc, upd_mil, all_miles, id, success = {}, [], [], [], []
+    target_mil = fragment_retrieval(gtx["projecta"], ["milestones"], milestone_id)
+    if target_mil:
+        for prum in target_mil:
+            milestones = prum['milestones']
+            for milestone in milestones:
+                if milestone.get('uuid')[0:len(milestone_id)] == milestone_id:
+                    upd_mil.append(milestone)
+                else:
+                    all_miles.append(milestone)
+            if upd_mil:
+                pid = prum.get('_id')
+                id.append(pid)
+    if len(upd_mil) == 0:
```

Comment:
maybe handle this at the top?

Reply:
included in an else statement that completes the if statement: `if target_mil` so that it is handled at the top.